Add "Snapchat clarifies activity tracking for personalising ads" memo (#553)
ambanum/OpenTermsArchive.org: content/memos/snapchat-clarifies-activity-tracking-for-personalising-ads/index.md

---
html_description: "Snapchat narrowed the scope of its ad personalisation tracking, limiting interest inferences to friends rather than household members, and excluding gender, interest, and friends' interest cohorts for teen accounts in the EU, UK, Norway, and Switzerland."
title: Snapchat clarifies activity tracking for personalising ads
service: Snapchat
terms_types: ["Privacy Policy"]
dates: ["2026-04-03"]
author: ["Saumyaa Naidu"]
related_collections: ["vlopses-us"]
---

> In the Snapchat Ads Transparency policy, the platform [__removed__](https://github.com/OpenTermsArchive/vlopses-us-versions/commit/87e81b683b2a4f7ea2be8d6b1cea74d9634deac9) collection of information about similar interests from “Snapchatters” in the same household. Snap will only collect this information from “friends” of the user now. It uses information about their interactions with ads or content to determine whether to show those ads to the user. Snapchat also clarified it won't use inferences about gender cohort, interests, or friends' interests (previously "interests of other people associated with") teen accounts in the EU, UK, Norway, and Switzerland. These changes are part of Snapchat's broader activity tracking, which involves making inferences for ad personalisation.

Reviewer: ```suggestion
In its Privacy Policy, Snapchat [removed](https://github.com/OpenTermsArchive/vlopses-us-versions/commit/87e81b683b2a4f7ea2be8d6b1cea74d9634deac9) the use of similar information from “Snapchatters” in the same household. Snapchat will only use this information from “friends” of the user now. It uses information about their interactions with ads or content to determine whether to show those ads to the user. Snapchat also clarified it won't use inferences about gender cohort, interests, or friends' interests (previously "interests of other people associated with you") for teen accounts in the EU, UK, Norway, and Switzerland. These changes are part of Snapchat's broader activity tracking, which involves making inferences for ad personalisation.
```
